AwsCryptographicMaterialProviders/runtimes/python/src/aws_cryptographic_material_providers/internaldafny/generated/dafny_src-py.dtr

```diff
@@ -1,238 +1,179 @@
 file_format_version = "1.0"
-dafny_version = "4.9.2.0"
+dafny_version = "4.9.0.0"
```

Comment:
Is this intentional? OH this is a dtr file, NVM.

Reply:
If I had time, I would stop committing .dtr files as part of this release. These are only used for one test scenario during the release process, and there's a way to execute this scenario without needing to track dtr files.